sleuth_backend/solr_module/solr_module.py

```diff
@@ -0,0 +1,11 @@
+import pysolr
+
+solr_url = 'http://localhost:8983/solr/'
+solr_core = "test"
```

Comment:
Might want to change this to 'default_core' later because we will probably end up having multiple cores. This is fine for now, though.